webrtc/RTCDataChannel-send.html

```diff
@@ -0,0 +1,380 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCDataChannel.prototype.send</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="RTCPeerConnection-helper.js"></script>
+<script>
+  'use strict';
+
+  // Test is based on the following editor draft:
+  // https://w3c.github.io/webrtc-pc/archives/20170605/webrtc.html
+
+  // The following helper functions are called from RTCPeerConnection-helper.js:
+  //  createDataChannelPair
+  //  awaitMessage
+  //  blobToArrayBuffer
+  //  assertEqualsArrayBuffer
```

Comment:
As much as I truly wish that the convention for assertion functions was camel-case, this breaks consistency with all of testharness.js and all of your own assertion functions.